build-tools/packages/build-cli/src/commands/modify/fluid-imports.ts

```diff
@@ -3,7 +3,7 @@
  * Licensed under the MIT License.
  */
 
-/* eslint-disable max-depth */
+/* eslint-disable unicorn/no-array-callback-reference */
```

Comment:
Curious why this needed to be disabled (not blocking, pure curiosity).

Reply:
This one is a little silly as far as I know.
If you have a function that is good use in a callback for like in .filter or .map, that rule is violated. It wants you to do `.filter((v) => foo(v))` instead of simple `.filter(foo)`. There were just too many places I was hitting it.